### PR TITLE
LPD-15209 - Update modules to ESM

### PR DESCRIPTION
**Pull request by bryceosterhaus**
another batch. testing CI

This is based on top of https://github.com/liferay-frontend/liferay-portal/pull/3972

### CODE REVIEW

**Comment by bryceosterhaus** [comment]
ci:test:relevant

**Comment by bryceosterhaus** [comment]
ci:test:relevant

**Comment by bryceosterhaus** [comment]
ci:test:sf

**Comment by bryceosterhaus** [comment]
ci:report:2534872452

**Comment by bryceosterhaus** [comment]
ci:test:relevant

**Comment by bryceosterhaus** [comment]
ci:report:2535230292

**Comment by bryceosterhaus** [comment]
ci:test:frontend-ui

**Comment by bryceosterhaus** [comment]
ci:test:sf

**Comment by bryceosterhaus** [comment]
ci:stop

**Comment by bryceosterhaus** [comment]
ci:test:sf

**Comment by bryceosterhaus** [comment]
ci:test:relevant

File: modules/apps/portlet-configuration/portlet-configuration-css-web/src/main/resources/META-INF/resources/js/index.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ import React, {useState} from 'react';
 
 const HEX_COLOR_REGEX = /^#?[0-9A-F]{3}(?:[0-9A-F]{3})?$/i;
 
-export function ColorPicker({color, label, name}) {
+export function ColorPickerInput({color, label, name}) {
```
Comment (victorg1991) [inline modules/apps/portlet-configuration/portlet-configuration-css-web/src/main/resources/META-INF/resources/js/index.js:11]: I have sent a fix for this: https://github.com/liferay-echo/liferay-portal/pull/15147 should I close the PR? or we can remove this commit?

Comment (bryceosterhaus) [reply modules/apps/portlet-configuration/portlet-configuration-css-web/src/main/resources/META-INF/resources/js/index.js:11]: @victorg1991 feel free to keep your PR open as it'll likely get merged first. I will resolve conflicts before sending to brian

**Comment by bryceosterhaus** [comment]
ci:report:2535898912

**Comment by bryceosterhaus** [comment]
ci:forward:force

**Comment by bryceosterhaus** [comment]
ci:stop

**Comment by bryceosterhaus** [comment]
manually sent https://github.com/brianchandotcom/liferay-portal/pull/147055

**Pull request closed without merging**